l10n_it_sdi_channel/views/company_view.xml

```diff
@@ -32,6 +32,15 @@
                                 </div>
                             </div>
                         </div>
+                        <div class="o_setting_left_pane">
+                            <field name="group_sdi_channel_validate_send"/>
+                        </div>
+                        <div class="o_setting_right_pane">
+                            <label for="group_sdi_channel_validate_send"/>
+                            <div class="text-muted">
+                                Allow users to validate, export and send invoices to SdI in one click.
```

Comment:
Qui non è proprio "Allow users" perchè se faccio click qui sto abilitando il gruppo per me, quindi penso sia meglio solo "validate, export and send invoices to SdI in one click"